docs/roadmap.md

```diff
@@ -7,78 +7,78 @@ The following roadmap gives an overview of what we are currently working on and
 
 ## Current focus
 
-**Search & Resource page redesign**<br/>
+#### **Search & Resource page redesign**<br/>
```

Comment:
do we still need <br/> here?

Reply:
No, will clean it up. We also don't need to use ** to bold format as bold

Reply:
still here?